.changeset/large-icons-complain.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+Expose standard Request object to endpoints
```

Comment:
Do we want this to say 'to endpoints and to hooks'? Or something similar to that?